### ISSUE TITLE
Unable to install packages to Ubuntu

### ISSUE THREAD

**Issue by vpolikarpov-akvelon** [adoptium/installer#754]
I'm trying to install different versions of Temurin JDK on Ubuntu 20.04 LTS. I've added repository following instructions provided in Artifactory and successfully received  packages list running `apt-get update`, but an error emerges when trying to install package. It varies from time to time, but it always somewhat related with access policies. Below are a couple of examples.

```
Err:1 https://packages.adoptium.net/artifactory/deb focal/main amd64 temurin-11-jdk amd64 11.0.20.1.0+1
  403  Forbidden [IP: 146.75.39.42 443]
E: Failed to fetch https://packages.adoptium.net/artifactory/deb/pool/main/t/temurin-11/temurin-11-jdk_11.0.20.1.0+1_amd64.deb  403  Forbidden [IP: 146.75.39.42 443]
```
```
Err:1 https://packages.adoptium.net/artifactory/deb jammy/main amd64 temurin-17-jdk amd64 17.0.8.1.0+1
  Error reading from server. Remote end closed connection [IP: 146.75.39.42 443]
E: Failed to fetch https://packages.adoptium.net/artifactory/deb/pool/main/t/temurin-17/temurin-17-jdk_17.0.8.1.0%2b1_amd64.deb  Error reading from server. Remote end closed connection [IP: 146.75.39.42 443]
```

I tried to access those URLs from browser and got the same result: nothing downloads, there are only errors. What may be the cause for this behavior?

**Comment by urskog84** [comment]
Havnig samular issue when install ver **17.0.8+1** on Ubuntu 2204 

I'm using packer to build a runner-mage (https://github.com/actions/runner-images)

```    
    azure-arm.build_image: The following NEW packages will be installed:
    azure-arm.build_image:   temurin-17-jdk
    azure-arm.build_image: 0 upgraded, 1 newly installed, 0 to remove and 28 not upgraded.
    azure-arm.build_image: Need to get 165 MB of archives.
    azure-arm.build_image: After this operation, 330 MB of additional disk space will be used.
    azure-arm.build_image: Get:1 https://packages.adoptium.net/artifactory/deb jammy/main amd64 temurin-17-jdk amd64 17.0.8.1.0+1 [165 MB]
    azure-arm.build_image: Ign:1 https://packages.adoptium.net/artifactory/deb jammy/main amd64 temurin-17-jdk amd64 17.0.8.1.0+1
    azure-arm.build_image: Err:1 https://packages.adoptium.net/artifactory/deb jammy/main amd64 temurin-17-jdk amd64 17.0.8.1.0+1
    azure-arm.build_image:   Error reading from server. Remote end closed connection [IP: 146.75.119.42 443]
==> azure-arm.build_image: E: Failed to fetch https://packages.adoptium.net/artifactory/deb/pool/main/t/temurin-17/temurin-17-jdk_17.0.8.1.0%2b1_amd64.deb  Error reading from server. Remote end closed connection [IP: 146.75.119.42 443]
==> azure-arm.build_image: E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
==> azure-arm.build_image: cat: /usr/lib/jvm/temurin-17-jdk-amd64/release: No such file or directory
```

**Comment by urskog84** [comment]
Seams like it works now 👍

**Comment by karianna** [comment]
@vpolikarpov-akvelon Can you try again as well?  It could have been a glitch in our CDN or the underlying JFrog repo.

**Comment by vpolikarpov-akvelon** [comment]
Looks good now, thank you!

**Comment by lhotari** [comment]
The packaging issue was resolved, but I seem to be hitting this kind of issue once in a while in the Apache Pulsar CI builds on GitHub Actions:
```
[INFO] DOCKER> Err:6 https://packages.adoptium.net/artifactory/deb jammy/main amd64 temurin-17-jdk amd64 17.0.9.0.0+9
  Error reading from server. Remote end closed connection [IP: 146.75.107.42 443]
[INFO] DOCKER> Fetched 5654 kB in 3s (2129 kB/s)
[INFO] DOCKER> [91mE: Failed to fetch https://packages.adoptium.net/artifactory/deb/pool/main/t/temurin-17/temurin-17-jdk_17.0.9.0.0%2b9_amd64.deb  Error reading from server. Remote end closed connection [IP: 146.75.107.42 443]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```
